CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing to Roslyn
+
+Guidelines for contributing to the Roslyn repo.
+
+## Submitting Pull Requests
+
+- **DO** ensure submissions pass all Jenkins legs and are merge conflict free.
+- **DO** ensure submissions meet functional and performance expectations, including scenarios for which the team doesn’t yet have open source tests. This means you may be asked to fix and resubmit your pull request against a new open test case if it fails one of these tests.
```

Comment:
Glad someone else felt that way too.  This is a hold over from our original attempt at documenting our processes.  Given the very low number of times we've had to do this I think it's fine to remove.